src/main/java/io/stargate/sgv2/jsonapi/service/schema/tables/CQLSAIIndex.java

```diff
@@ -144,6 +152,16 @@ static IndexTarget indexTarget(IndexMetadata indexMetadata)
       columnName = matcher.group(2);
       functionName = matcher.group(1);
     }
+    // At this point, after matcher, we may get a columnName doubleQuoted string from the index
+    // target
+    // E.G. 'target': 'values("a_list_column")' -> "a_list_column", 'target':
+    // 'values("a_regular_column")' -> "a_regular_column"
+    // 1. We can NOT use fromInternal which will keep the double quote in the identifier
+    // 2. We can NOT use fromCQL, since it will lowercase the unquoted string, E.G. 'BigApple' ->
+    // bigapple
+    // So we should just stripe the doubleQuote if needed
+    columnName =
+        Strings.isDoubleQuoted(columnName) ? Strings.unDoubleQuote(columnName) : columnName;
```

Comment:
Looks like `unDoubleQuote()` immediate checks `isDoubleQuoted()` so can simplify as:

    columnName = Strings.unDoubleQuote(columnName);

and works the same, but avoids calling `isDoubleQuoted()` twice.

Reply:
thanks, solved!